dashboard/db/migrate/20170331235243_create_experiments.rb

```diff
@@ -0,0 +1,19 @@
+class CreateExperiments < ActiveRecord::Migration[5.0]
+  def change
+    create_table :experiments do |t|
```

Comment:
Do we want to make the timestamp columns all in `at`? E.g., `start_at`, `end_at`, `earliest_section_start_at`, `latest_section_start_at`?